model/common/src/icon4py/model/common/utils/gt4py_field_allocation.py

```diff
@@ -25,7 +25,7 @@ def allocate_zero_field(
     if is_halfdim:
         assert len(shapex) == 2
         shapex = (shapex[0], shapex[1] + 1)
-    return gtx.as_field(dims, xp.zeros(shapex, dtype=dtype), allocator=backend)
+    return gtx.as_field(dims, np.zeros(shapex, dtype=dtype), allocator=backend)
```

Comment:
here you could directly use the `gt4py` function: 
```suggestion
def allocate_zero_field(
    *dims: gtx.Dimension,
    grid,
    is_halfdim=False,
    dtype=ta.wpfloat,
    backend: Optional[backend.Backend] = None,
):
    
    def size(dim:gtx.Dimension, is_half_dim: bool) -> int:
         if dim == dims.KDim and is_half_dim:
            return grid.size[dim] + 1
         else:
            return  grid.size[dim]
        
    dimensions = {d: range(size(d, is_halfdim)) for d in dims }
    return gtx.zeros(dimensions, dtype=dtype, allocator=backend)


```

Reply:
Super thanks!
I did not know that how I could properly create a new dimension until now...